public/templates/tracks/show.html

```diff
@@ -22,6 +22,7 @@
             <li>Plays: <strong>{{::track.stats.plays}}</strong></li>
             <li>Downloads: <strong>{{::track.stats.downloads}}</strong></li>
             <li>Favourites: <strong>{{::track.stats.favourites}}</strong></li>
+            <li><strong><a href="{{::track.url}}/stats">View more stats</a></strong></li>
```

Comment:
A route needs to exist in `routes.php` for this so the URL doesn't 404 if one clicks a link directly to the stats page.